app/workers/package_manager_download_worker.rb

```diff
@@ -1,10 +1,15 @@
+# frozen_string_literal: true
+
 class PackageManagerDownloadWorker
   include Sidekiq::Worker
   sidekiq_options queue: :critical
 
-  def perform(class_name, name)
-    return unless class_name.present?
-    logger.info("Beginning update for #{class_name}/#{name}")
-    "PackageManager::#{class_name}".constantize.update(name)
+  def perform(clazz, name)
+    return unless clazz.present?
+
+    # need to maintain compatibility with things that pass in the name of the class under PackageManager module
+    clazz = "PackageManager::#{clazz}".constantize if clazz.is_a? String
```

Comment:
I think the only two spots calling this with a String are

https://github.com/librariesio/libraries.io/blob/46c85a9e730e25e279267e8dcbba307c46916942/lib/tasks/projects.rake#L112

https://github.com/librariesio/libraries.io/blob/4e733b309780632cc81616dc1c84838f5f0b42f3/app/controllers/hooks_controller.rb#L17